crates/gpui/src/app.rs

```diff
@@ -663,6 +663,10 @@ impl AppContext {
                     }
                 }
             } else {
+                //todo!(linux): this `cx.draw()` command
+                // resets the `dirty` flag, thus preventing
+                // Linux presentation from kicking off.
+                #[cfg(not(target_os = "linux"))]
```

Comment:
Going to dig into this change, we need to support 'Flush effects loops always end in a draw' in tests so that Zed tests which rely on stateful rendering can be implemented.

Reply:
omg, this was caused by the livekit client compiling GPUI in test mode 🤦🏻‍♀